vme/src/dilexp.cpp

```diff
@@ -6779,7 +6779,7 @@ void dilfe_hrt(dilprg *p)
 
     v->type = DILV_SINT2R;
     v->atyp = DILA_NONE;
-    v->ref = (sbit16 *)&(p->sarg->fptr->heart_beat);
+    v->ref = p->sarg->fptr->getHeartBeatPtr();
```

Comment:
Wonder if there's a way to catch it, if someone changes the Heartbeat type in the class. That'd break this code here because the v->type would need to be updated.